app/src/constants/paths.js

```diff
@@ -2,3 +2,4 @@ import {normalize} from 'path'
 
 export const lib = normalize(`${__dirname}/..`)
 export const root = normalize(`${lib}/../..`)
+export const chat = 'chat'
```

Comment:
do you mean add an utility function to `app/src/app/utils.js` and use that in `app/src/app/close.js` instead right?